src/main/java/gregtech/api/GregTechAPI.java

```diff
@@ -62,10 +63,18 @@ public class GregTechAPI {
 
     public static final BaseCreativeTab TAB_GREGTECH =
             new BaseCreativeTab(GTValues.MODID + ".main", () -> MetaItems.BATTERY_HULL_HV.getStackForm(), true);
+    public static final BaseCreativeTab TAB_GREGTECH_MACHINES =
+            new BaseCreativeTab(GTValues.MODID + ".machines", () -> MetaTileEntities.ELECTRIC_BLAST_FURNACE.getStackForm(), true);
+    public static final BaseCreativeTab TAB_GREGTECH_CABLES =
+            new BaseCreativeTab(GTValues.MODID + ".cables", () -> OreDictUnifier.get(OrePrefix.wireGtDouble, Materials.Aluminium), true);
```

Comment:
This should use the cable instead of the wire. It looks somewhat similar to the pipe, which is in the next tab, which could cause confusion.